### ISSUE TITLE
Make "Wavebox is checking for updates" more accurate

### ISSUE THREAD

**Issue by VarunAgw** [wavebox/waveboxapp#1027]
When it found an update, it immediately start downloading, but doesn't update the displayed text.

It would be nicer if it could say "Found update.. downloading" or something like that.

**Comment by nmat** [comment]
You mean you want some progressbar for the update?

**Comment by VarunAgw** [comment]
No. A textual feedback that program found updates and is downloading it will be good.

**Comment by nmat** [comment]
hmmm I get a dialog when there is a new update. Can you say what settings you have set for the updates and also what system you are on and also what type of package?

**Comment by VarunAgw** [comment]
@nmat I have disabled automatic updates and I click on "Check for updates".

Windows 10 x64,

**Comment by Thomas101** [comment]
This is a limitation of the upstream squirrel updater package at the moment. We get told when there is an update and then when it has been downloaded. There's quite a bit of disruptive activity upstream though, so I'll see if there's any scope for adding this in
